src/OrbitGl/TimeGraph.cpp

```diff
@@ -359,6 +359,31 @@ void TimeGraph::ProcessApiTrackValueEvent(const orbit_client_data::ApiTrackValue
   track->AddValue(time, track_event.value());
 }
 
+void TimeGraph::ProcessPresentEvent(const orbit_grpc_protos::PresentEvent& present_event) {
```

Comment:
I would expect this method to do something similar to the other `Process` methods (like `ApiTrackValue`), so just add the new timestamp to the needed Track. 

Better would be that this timestamp is added to the provider that doesn't have any relationship with the UI and the Track/s needed can get the data from the provider (as in ThreadTracks or ApiScope https://github.com/google/orbit/blob/main/src/OrbitGl/TimeGraph.cpp#L331).

Tracks after can get the needed data.